test/verify/naughty-debian-unstable/5399-storaged-crash-poll-with-variant

```diff
@@ -0,0 +1,6 @@
+Traceback (most recent call last):
+  File "./verify/check-storage-lvm2", line 155, in testLvm
```

Comment:
Because it's a vague and risky issue. I wanted to it to be more fragile than normal, requiring care and feeding, rather than accidentally cover too many failures long term.